PowerEditor/src/WinControls/ReadDirectoryChanges/ReadFileChanges.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+
+#include <stdio.h>
+
+#ifndef VC_EXTRALEAN
+#define VC_EXTRALEAN            // Exclude rarely-used stuff from Windows headers
+#endif
+
+
+#include <windows.h>
+
+
+class CReadFileChanges
+{
+public:
+	CReadFileChanges();
+	~CReadFileChanges();
+	void AddFile(LPCTSTR szDirectory, DWORD dwNotifyFilter);
+	BOOL CReadFileChanges::DetectChanges();
+	void Terminate();
+
+private:
+	LPCTSTR szFile;
```

Comment:
Member variables should be initialized either in class member initializer or in contructor.

Reply:
Solved. Thanks.